intranet/apps/signage/migrations/0015_auto_20180420_1825.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
As do these

Reply:
The build fails without it.

Reply:
Sure, but it should really be a separate PR as it's completely unrelated.